src/utils.py

```diff
@@ -0,0 +1,12 @@
+import base64
+
+from solana.publickey import PublicKey
+from solana.rpc.api import Client
+
+
+def load_bytes_data(addr: PublicKey, endpoint: str):
+    res = Client(endpoint).get_account_info(addr)
+    if not res:
```

Comment:
I think this is guaranteed. So the exception here is not useful. But you should check if the `result`, `value`, or `data` key is missing.